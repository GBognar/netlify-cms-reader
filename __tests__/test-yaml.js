const { getConfig, getData } = require('../index')

test('reads yaml data', async () => {
  const config = getConfig('__tests__/yaml/config.yml')
  const data = await getData(config)
  expect(data).toMatchSnapshot()
})
