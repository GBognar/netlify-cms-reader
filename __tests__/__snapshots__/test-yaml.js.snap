// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reads yaml data 1`] = `
Object {
  "layout": Object {
    "footer": Object {
      "content": "Footer Content",
      "copyright": "Â© 2019 Copyright",
    },
    "navbar": Object {
      "title": "Navbar Title",
    },
  },
  "pages": Array [
    Object {
      "content": "# About Page Content",
      "path": "/about",
      "title": "About Page",
    },
    Object {
      "content": "# Index Page Content",
      "path": "/",
      "title": "Index Page",
    },
  ],
}
`;
